spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/src/main/java/org/springframework/cloud/dataflow/tasklauncher/CannotHandleRequestException.java

```diff
@@ -0,0 +1,6 @@
+package org.springframework.cloud.dataflow.tasklauncher;
```

Comment:
Need to add copyright and brief javadoc for this exception.

Reply:
Done